lib/sdk/tabs/observer.js

```diff
@@ -39,14 +39,28 @@ const Observer = Class({
     // that before emitting "activate" on selected tab.
     this.on("select", tab => {
       const selected = this[selectedTab];
+
+      // Bug 942511: this[selectedTab] is never updated when a popup window is
+      // closed. This leads to the situation where the properties of ActiveTab 
+      // are undefined because ActiveTab relates to the closed popup window. To 
```

Comment:
what exactly is `ActiveTab` in this comment?  is it the `tab` variable? the result of `getActiveTab()`? the `selected` variable or something else?